jobbergate-cli/jobbergate_cli/main.py

```diff
@@ -68,27 +66,8 @@ def main(
     init_logs(verbose=verbose)
     init_sentry()
 
-    protocol = "https" if settings.OIDC_USE_HTTPS else "http"
-    authentication_handler = JobbergateAuthHandler(
-        cache_directory=settings.JOBBERGATE_USER_TOKEN_DIR,
-        login_domain=f"{protocol}://{settings.OIDC_DOMAIN}",
-        login_audience=settings.OIDC_AUDIENCE,
-        login_client_id=settings.OIDC_CLIENT_ID,
-        login_client_secret=settings.OIDC_CLIENT_SECRET,
-        login_url_handler=show_login_message,
-        login_sequence_handler=track_login_progress,
-    )
-
-    ctx.obj = JobbergateContext(
-        client=httpx.Client(
-            base_url=settings.ARMADA_API_BASE,
-            auth=authentication_handler,
-            timeout=settings.JOBBERGATE_REQUESTS_TIMEOUT,
-        ),
-        authentication_handler=authentication_handler,
-        full_output=full,
-        raw_output=raw,
-    )
+    context: ContextProtocol = JobbergateContext(full_output=full, raw_output=raw)
+    ctx.obj = context  # local variable to enable type checking and make mypy happy with the syntax
```

Comment:
This also binds the JobbergateContext to the typer command context. It's not just for type-checking and mypy.

Reply:
The intention is to highlight that mypy is not happy if we make it directly:
```python
    ctx.obj: ContextProtocol = JobbergateContext(full_output=full, raw_output=raw))
```

For me, using `# type: ignore` is not an option as it would disable the type checking I intend to enforce.

I reworded the comment to clarify it: 894e6077f4ff2a0fe39a38ad50f6d87421af0bab